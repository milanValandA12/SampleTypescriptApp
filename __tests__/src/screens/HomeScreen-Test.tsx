import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import store from '../../../src/redux/store';
import HomeScreen from '../../../src/screens/main/home/home.screen';

describe('Home screen', () => {
    it('renders correctly', () => {
        const tree = renderer.create(
            <Provider store={store}>
                <HomeScreen />
            </Provider>
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
});