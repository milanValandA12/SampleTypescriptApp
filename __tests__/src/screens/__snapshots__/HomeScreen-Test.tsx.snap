// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home screen renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "padding": 8,
    }
  }
>
  <View>
    <TextInput
      onChangeText={[Function]}
      placeholder="common:search"
      style={
        Object {
          "borderColor": "#000000",
          "borderRadius": 12,
          "borderWidth": 1,
          "height": 50,
          "marginBottom": 12,
          "padding": 8,
        }
      }
      value=""
    />
    <RCTScrollView
      getItem={[Function]}
      getItemCount={[Function]}
      keyExtractor={[Function]}
      onContentSizeChange={[Function]}
      onLayout={[Function]}
      onMomentumScrollBegin={[Function]}
      onMomentumScrollEnd={[Function]}
      onRefresh={[Function]}
      onScroll={[Function]}
      onScrollBeginDrag={[Function]}
      onScrollEndDrag={[Function]}
      refreshControl={
        <RefreshControlMock
          onRefresh={[Function]}
          refreshing={false}
        />
      }
      refreshing={false}
      removeClippedSubviews={false}
      renderItem={[Function]}
      scrollEventThrottle={50}
      stickyHeaderIndices={Array []}
      viewabilityConfigCallbackPairs={Array []}
    >
      <RCTRefreshControl />
      <View />
    </RCTScrollView>
  </View>
</View>
`;
